From 0cab1559c4cc56dd1152fca2ebc309239fc76c03 Mon Sep 17 00:00:00 2001
From: Gregory CLEMENT <gregory.clement@free-electrons.com>
Date: Tue, 17 May 2016 11:28:04 +0200
Subject: [PATCH 1740/2241] arm64: dts: marvell: add tbg clocks for Armada 37xx

Add a new block of clocks. The Time Base Generators clocks can be the
parent of the peripheral clocks.

Change-Id: Ieaba169e7c2ee8e5f2f661287a8bf33bef2be873
Signed-off-by: Gregory CLEMENT <gregory.clement@free-electrons.com>
Signed-off-by: allen yan <yanwei@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40821
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 1e0b16f..2628645 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -295,6 +295,14 @@
 				clocks = <&sgateclk 4>;
 				status = "disabled";
 			};
+
+			tbg: tbg@13200 {
+				compatible = "marvell,armada-3700-tbg-clock";
+				reg = <0x13200 0x100>;
+				clocks = <&xtalclk>;
+				#clock-cells = <1>;
+			};
+
 			gpio_nb: gpio@13800 {
 				compatible = "marvell,armada3700-gpio",
 				"syscon", "simple-mfd";
-- 
2.7.4

